tests/sentry/backup/test_roundtrip.py

```diff
@@ -75,7 +75,7 @@ def test_good_continuing_sequences(tmp_path):
     )
 
     # Empty the database without resetting primary keys.
-    clear_database_but_keep_sequences()
+    clear_database(reset_pks=True)
```

Comment:
is the comment accurate here?

Reply:
The comment is accurate, the code was not. I've removed this parameter - good catch!